README.md

```diff
@@ -44,6 +44,30 @@ Keras 2.1.5
 Tensorflow 1.7.0
 sklearn 0.18.1
```

Comment:
I'd suggest just removing this section, as it duplicates the information in the requirements.txt file.

Then, the section you're adding can go up one heading level (`###` instead of `####`) and maybe we should rename to just "Installing dependencies"?

Reply:
Yeah, that would be better to prevent confusion. Thanks!